01-data-structures/01-introduction-to-data-structures/screen/screen.rb

```diff
@@ -6,18 +6,23 @@ class Screen
   attr_accessor :matrix
 
   def initialize(width, height)
+    @width = width
+    @height = height
+    @matrix = Array.new(@height){Array.new(@width)}
```

Comment:
You *could* also use a Matrix object, but I think it's probably easier to do what you've done here and use arrays. Good use of the width and height to initialize.